edit/buildozer.go

```diff
@@ -1321,6 +1329,53 @@ func findBuildFiles(rootDir string) []string {
 	return buildFiles
 }
 
+// getIgnoredPrefixes returns a list of ignored prefixes from the .bazelignore file in the root directory.
+// It returns an empty list if the file does not exist.
+func getIgnoredPrefixes(rootDir string) []string {
+	bazelignorePath := filepath.Join(rootDir, ".bazelignore")
+	ignoredPaths := []string{}
+
+	data, err := os.ReadFile(bazelignorePath)
+	if err != nil {
+		return ignoredPaths
+	}
+
+	scanner := bufio.NewScanner(bytes.NewReader(data))
+	lineNum := 0
+
+	for scanner.Scan() {
+		lineNum++
+		line := strings.TrimSpace(scanner.Text())
+
+		// Skip empty lines, comments, and absolute paths
+		// Bazel will error out if there are any absolute paths in the .bazelignore file.
+		if line == "" || strings.HasPrefix(line, "#") || path.IsAbs(line) {
+			continue
+		}
+
+		ignoredPaths = append(ignoredPaths, line)
+	}
+
+	return ignoredPaths
+}
+
+// shouldIgnorePath returns true if the path should be ignored based on the list of ignored prefixes.
+func shouldIgnorePath(path string, rootDir string, ignoredPrefixes []string) bool {
+	rel, err := filepath.Rel(rootDir, path)
+	if err != nil {
+		return false
+	}
+	// Normalize path separators to forward slashes
+	rel = filepath.ToSlash(rel)
+
+	for _, prefix := range ignoredPrefixes {
+		if strings.HasPrefix(rel, prefix) {
```

Comment:
Will `foobar` be ignored if `foo` is among ignored prefixes?

Reply:
Ah yes, good call. I changed the comparison to only accept exact match or match with `prefix+"/"`, and changed `getIgnoredPaths` to `Clean` the paths so that `/` suffixes in .bazelignore are stripped from it. 111bc6e962b5115c91e5d9a0001910bdc423d6eb